samples/peripheral/802154_phy_test/src/ctrl/ptt_zb_perf_cmd_mode_response.c

```diff
@@ -0,0 +1,524 @@
+
+	ptt_uart_send_packet(evt_data->arr,
+			 strlen(PTT_CAST_TO_STR(evt_data->arr)));
+}
+
+void cmd_uart_send_rsp_get_dcdc(ptt_evt_id_t evt_id)
+{
+	struct ptt_evt_data_s *evt_data = ptt_event_get_data(evt_id);
+
+	assert(evt_data != NULL);
+
+	struct ptt_evt_ctx_data_s *ctx_data = ptt_event_get_ctx_data(evt_id);
+
+	assert(ctx_data != NULL);
+
+	sprintf(PTT_CAST_TO_STR(evt_data->arr), PTT_GET_DCDC_MES "%d", ctx_data->arr[0]);
+
+	ptt_uart_send_packet(evt_data->arr,
+			 strlen(PTT_CAST_TO_STR(evt_data->arr)));
+}
+
+void cmd_uart_send_rsp_get_temp(ptt_evt_id_t evt_id)
+{
+	struct ptt_evt_data_s *evt_data = ptt_event_get_data(evt_id);
+
+	assert(evt_data != NULL);
+
+	struct ptt_evt_ctx_data_s *ctx_data = ptt_event_get_ctx_data(evt_id);
+
+	assert(ctx_data != NULL);
+
+	sprintf(PTT_CAST_TO_STR(evt_data->arr),
+		"%d", *(int32_t *)ctx_data->arr);
+
+	ptt_uart_send_packet(evt_data->arr,
+			 strlen(PTT_CAST_TO_STR(evt_data->arr)));
+}
+
+void cmd_uart_send_rsp_antenna_error(ptt_evt_id_t evt_id)
+{
+	ptt_uart_send_packet(PTT_CAST_TO_UINT8_P(PTT_GET_ANTENNA_ERROR_MES),
+			 sizeof(PTT_GET_ANTENNA_ERROR_MES) - 1);
+}
+
+void cmd_uart_send_rsp_antenna(ptt_evt_id_t evt_id)
+{
+	struct ptt_evt_data_s *evt_data = ptt_event_get_data(evt_id);
+
+	assert(evt_data != NULL);
+
+	struct ptt_evt_ctx_data_s *ctx_data = ptt_event_get_ctx_data(evt_id);
+
+	assert(ctx_data != NULL);
+
+	sprintf(PTT_CAST_TO_STR(evt_data->arr), PTT_GET_ANTENNA_MES "%d", ctx_data->arr[0]);
+
+	ptt_uart_send_packet(evt_data->arr,
+			 strlen(PTT_CAST_TO_STR(evt_data->arr)));
+}
+
+static void cmd_uart_send_rf_packet_with_num(ptt_evt_id_t new_rf_pkt_evt, uint32_t packet_n)
+{
+	struct ptt_evt_data_s *rf_evt_data = ptt_event_get_data(new_rf_pkt_evt);
+
+	assert(rf_evt_data != NULL);
+
+	struct ptt_evt_ctx_data_s *rf_ctx_data = ptt_event_get_ctx_data(new_rf_pkt_evt);
+
+	assert(rf_ctx_data != NULL);
+
+	char                   out_str[CMD_UART_BUF] = {0};
+	struct ptt_rf_packet_info_s *pkt_info =
+		(struct ptt_rf_packet_info_s *)(rf_ctx_data->arr);
+
+	sprintf(out_str,
+		"C:%d L:%d Data:",
+		packet_n,
+		rf_evt_data->len);
+
+	uint16_t filled_length = strlen(out_str);
+
+	/* we have each byte coded as two ASCII symbols */
+	if ((rf_evt_data->len * 2) < (CMD_UART_BUF - filled_length)) {
+		for (int i = 0; i < rf_evt_data->len; ++i) {
+			uint8_t lst = rf_evt_data->arr[i] & 0x0F;
+			uint8_t mst = (rf_evt_data->arr[i] >> 4) & 0x0F;
+
+			out_str[filled_length + 2 * i]     =
+				(mst >= 0xA) ? mst - 0xA + 'A' : mst + '0';
+			out_str[filled_length + 2 * i + 1] =
+				(lst >= 0xA) ? lst - 0xA + 'A' : lst + '0';
+		}
+
+		/* let's use rf_evt_data as temporary buffer, as we copied everything from there */
+		sprintf(PTT_CAST_TO_STR(rf_evt_data->arr),
+			" CRC:1 LQI:0x%02x RSSI:%d",
+			pkt_info->lqi,
+			pkt_info->rssi);
+
+		/* concatenate everything together */
+		if ((filled_length + rf_evt_data->len * 2 +
+			strlen(PTT_CAST_TO_STR(rf_evt_data->arr)) +
```

Comment:
This looks really bad the contents of the `if` condition blends with the actual code block - we typically align to the opening bracket. There are also multiple other places like this accross the code base, `clang-format` would've handed these if applied as suggested.

Reply:
Fixed by executing clang-format on all .c and .h files in the sample.